ci/deploy.sh

```diff
@@ -198,6 +214,10 @@ EOF
         install -Dsm755 -t pkg/debian/usr/bin target/release/strip-ansi-escapes
 
         deps=$(cd pkg && dpkg-shlibdeps -O -e debian/usr/bin/*)
+	      mv pkg/debian/postinst pkg/debian/DEBIAN/postinst
```

Comment:
indent looks a bit weird here; tabs vs. spaces?

Reply:
It's my editor. It looked different everywhere. I tried to fix it on the github editor but apparently it's still borked. Sorry.